### ISSUE TITLE
Importance Ambiguity

### ISSUE THREAD

**Issue by satabios** [VainF/Torch-Pruning#329]
I'm trying to prune the model channelwise and I'm using a torch.norm['Frobenius Norm'] for importance; while I try to replicate the same with 
`tp.importance.MagnitudeImportance(p=2, target_types=[torch.nn.Conv2d])`. The results are drastically different. Am I missing something? If not how would I go about it?

Please find the below norm calculation I made:

      def get_input_channel_importance(self, weight):
         
  
          in_channels = weight.shape[1]
          importances = []
          # compute the importance for each input channel
          for i_c in range(weight.shape[1]):
              channel_weight = weight.detach()[:, i_c]        
              importance = torch.norm(channel_weight)
              importances.append(importance.view(1))
          return torch.cat(importances)

**Comment by sidsingla** [comment]
Hi,
Were you able to figure out the issue?

Thanks!
